desktop/app/lib/config/index.js

```diff
@@ -8,13 +8,21 @@ config.version = pkg.version;
 config.author = pkg.author;
 
 config.loginURL = function () {
+	if ( process.env.WP_DESKTOP_BASE_URL ) {
```

Comment:
I'm getting some weird things happening when logged out or when using the Home button.
I get the error `(HTTP code 400) unexpected - invalid reference format`
In these cases it is trying to load URLs like `https://calypso.live/?image=registry.a8c.com/calypso/app:build-%%%%%/log-in` and `https://calypso.live/?image=registry.a8c.com/calypso/app:build-%%%%%/stats/day` which break.

I think we may have to convert the URLs to container URLs like in `.teamcity/_self/projects/WebApp.kt` as well

Reply:
I can probably update the TC comment code to use the correct container URL. That way, users don't have to interact with the image redirect thing. Then we don't necessarily have to handle it here